app/indexing.py

```diff
@@ -208,8 +210,29 @@ def sort_query_words(query_words):
     #       which are usually more effective filters, come first.
     return sorted(sorted_query_words, key=len, reverse=True)
 
-
-def search(repo, query_obj, max_results):
+def create_query(query):
+    return re.sub(r' ', ' AND ', query)
+
+def search_with_index(repo, query_obj, max_results):
```

Comment:
Let's move all the full text search/index related functions and logic into a new file e.g. full_text_search.py

It helps during the transition period, and make the separation clearer.